wagtailmedia/forms.py

```diff
@@ -78,3 +78,13 @@ def get_media_form(model):
     ],
     'wagtailmedia/permissions/includes/media_permissions_formset.html'
 )
+
+
+class MediaInsertionForm(forms.Form):
+    """
+    Form for customizing media player behavior (e.g. autoplay by default)
+    prior to insertion into a rich text area
+    """
+    autoplay = forms.BooleanField(required=False)
```

Comment:
I wouldn’t ever recommend anyone make anything autoplay, although I know some people might. How can we make the presence of this field configurable?